### DISCUSSION TITLE
Input of type Number only

### DISCUSSION THREAD

**Discussion by ghost** [quasarframework/quasar#9154] [CLI - PWA mode]
Hello all

am having an small issue with an input field I have set the type as number an its stops me to input alphabets but its allowing the letter 'e' to in the input 

please help 
Thanks 
[input.txt](https://github.com/quasarframework/quasar/files/6416820/input.txt)

**Comment by TobyMosque** [comment]
type is handled by the browser, and `input[type=number]` is specified to accept `floating-point`, what includes numbers like 12+3687e
if u wanna to prevent that behavior, u would use a mask (`reverse-fill-mask` to be exact).
